tools/legal-review/engine/com.lihaoyi.fansi_2.13-0.4.0/files-keep

```diff
@@ -1 +1 @@
-/com-lihaoyi/fansi/blob/master/LICENSE
+/com-lihaoyi/Fansi/blob/master/LICENSE
```

Comment:
Why are we changing upper case?

Reply:
This is because some license data was scraped from github and the website structure has changed, so we had to update the logic a little bit. It resulted in the case changing in a few places and I decided it is simplest to just update the config rather than modify the logic to be consistent with the 'old' way, as that would just have added complexity with no real purpose.